### ISSUE TITLE
UB in wrapper function signatures

### ISSUE THREAD

**Issue by JLockerman** [zombodb/pgx#241]
The `#[pg_extern]` function wrappers exposed to postgres are [currently defined](https://github.com/zombodb/pgx/blob/d9c1db535bb5bd27f6cb70217866a040f3f98efb/pgx-macros/src/rewriter.rs#L203) as
```rust
unsafe fn(*mut FunctionCallInfoBaseData) -> usize
```
while they should be defined as `extern "C"` functions like
```rust
unsafe extern "C" fn(*mut FunctionCallInfoBaseData) -> usize
```
without the specification the function [defaults to](https://doc.rust-lang.org/reference/items/functions.html#extern-function-qualifier) the [perma](https://github.com/rust-lang/rfcs/issues/600#issuecomment-526033825)-[unstable](https://github.com/rust-lang/rfcs/issues/600#issuecomment-526812431) `"Rust"` ABI which does not match the `"C"` ABI Postgres is expecting to call.

**Comment by eeeebbbbrrrr** [comment]
Ha. No way!?!? That’s incredibly embarrassing. Will get it resolved this week.

**Comment by eeeebbbbrrrr** [comment]
Got this fixed up here: https://github.com/zombodb/pgx/commit/756e5fdec8395be18e8203c35bb4a4383661ba9b

We'll get a 0.1.23 version out later today.

**Comment by eeeebbbbrrrr** [comment]
released in v0.1.23 and published to crates.io.  Thanks again, @JLockerman!
